### ISSUE TITLE
No releases are tagged

### ISSUE THREAD

**Issue by QuLogic** [ramnathv/htmlwidgets#290]
Several releases have been made to CRAN, but none of them have been tagged here. This makes it difficult to find the exact commit from which any release originated.

**Comment by timelyportfolio** [comment]
@ramnathv, @jcheng5 I agree that for stability and robustness, we should tag future releases on Github starting with current.  I am not sure there is much benefit in back-tagging releases.  Please let me know how I can help.  Thanks.
